packages/botonic-plugin-contentful/src/contentful/delivery-utils.ts

```diff
@@ -144,3 +145,22 @@ export function convertContentfulException(e: any, query: any): any {
   }
   return e
 }
+//supported types: string, number and boolean
+export function addCustomFields<T extends TopContent>(
+  content: T,
+  entryFields: CommonEntryFields,
+  ignoreFields?: string[]
+): T {
+  const customKeys = Object.keys(entryFields).filter(
+    f =>
+      !Object.keys(content).includes(f) &&
+      !Object.keys(content.common).includes(f) &&
+      !ignoreFields?.includes(f) &&
+      //contentful: followup, plugin: followUp
```

Comment:
remove console.log?

Reply:
This is info to indicate that it has to ignore 'followup' content name. We are receiving 'followup' from contentful but in the plugin is named followUp